crypto/keys/keybase_base.go

```diff
@@ -167,7 +167,7 @@ func (kb baseKeybase) CreateMnemonic(
 // Derive computes a BIP39 seed from the mnemonic and bip39Passphrase. It creates
 // a private key from the seed using the BIP44 params.
 func (kb baseKeybase) Derive(
-	keyWriter keyWriter, name, mnemonic, bip39Passphrase, encryptPasswd string, params hd.BIP44Params,
+	keyWriter keyWriter, name, mnemonic, bip39Passphrase, encryptPasswd string, params fmt.Stringer,
```

Comment:
ditto

Reply:
is there a need to pass hd.BIP44Params if you only use `.string()`, or is there a betteer way to do this?

Reply:
Pass the bip44Params string directly ?